deploy/resources.yml

```diff
@@ -83,6 +83,15 @@ Resources:
               Service: lambda.amazonaws.com
             Action: "sts:AssumeRole"
       Policies:
+        - PolicyName: ManifestPipelineSsmPermissions
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              -
+                Action:
+                  - "ssm:Get*"
+                Effect: "Allow"
+                Resource: "*"
```

Comment:
Since we know the path where we will be storing the config, we should scope the pipeline permissions down to only be able to read that one path. You may also want to scope the Action down to `ssm:GetParameter*` unless you think the Lambda will need to use any of the other `ssm:Get` commands.

Reply:
Permissions updated